Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:17.04
+
+MAINTAINER Obul <obulpathi@merkletree.vc>
+
+# force bash always
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+
+# install dependencies
+RUN apt-get update && apt-get install -y git libssl-dev libffi-dev python3.6 python3.6-dev python3-pip
+RUN pip3 install virtualenv
```

Comment:
Having a `virtualenv` inside a container slightly defeats the purpose. The point of the container is that it's already its own environment.